test/services/listCourtCases.integration.test.ts

```diff
@@ -651,6 +651,47 @@ describe("listCourtCases", () => {
     })
   })
 
+  describe.only("filter by cases allocated to me", () => {
+    it("should list cases that are locked to me", async () => {
+      const orgCode = "36FPA1"
+
+      await insertCourtCasesWithFields([
+        { errorLockedByUsername: "User1", triggerLockedByUsername: "User1", orgForPoliceFilter: orgCode },
```

Comment:
Maybe we could add a test for when only errors or triggers are locked by username?

Reply:
We've added 2 more tests.